soc/arm/nuvoton_npcx/npcx7/soc.c

```diff
@@ -42,3 +42,9 @@ NPCX_REG_SIZE_CHECK(uart_reg_t, 0x027);
 NPCX_REG_OFFSET_CHECK(uart_reg_t, UPSR, 0x00E);
 NPCX_REG_OFFSET_CHECK(uart_reg_t, UFTSTS, 0x020);
 NPCX_REG_OFFSET_CHECK(uart_reg_t, UFRCTL, 0x026);
+
+/* MIWU register structure check */
+NPCX_REG_SIZE_CHECK(miwu_reg_t, 0x078);
+NPCX_REG_OFFSET_CHECK(miwu_reg_t, LOW_PND_WU, 0x00B);
+NPCX_REG_OFFSET_CHECK(miwu_reg_t, WKPND6, 0x02E);
```

Comment:
Regarding lower case for hex value, I found lots of examples use upper case for hex value. Is it a requirement for code style?

Reply:
I don't think so (the coding style doesn't go to that level of detail).  I would probably use upper case for small constants like this, but lower case in larger values because it's more readable.

What's a bit odd here is the leading zero making it a 12-bit constant rather than an 8-bit one.  Is that intentional?